app/api/__init__.py

```diff
@@ -326,7 +326,8 @@
 # attendees
 api.route(AttendeeListPost, 'attendee_list_post', '/attendees')
 api.route(AttendeeList, 'attendee_list', '/events/<int:event_id>/attendees',
-          '/events/<event_identifier>/attendees', '/orders/<int:order_id>/tickets/<int:ticket_id>/attendees')
+          '/events/<event_identifier>/attendees', '/orders/<int:order_id>/attendees',
```

Comment:
I think the above `<int:order_id>` was missed in changing to order_identifier, can you confirm please.
Rest looks good.